strapi/strapi
# enhancement: move installed plugins page to settings

### What does it do?

Moved the installed plugins list page to the settings page

### How to test it?

- you shouldn't see the plugins link in the admin man nav
- you should see the plugins link in the settings page, in the general section
- when logged in as an author or editor, you shouldn't see the plugins link in the settings
- if you grant the permission for marketplace and plugins access to that role, then you should see the link again
